SIPS/sip-78.md

```diff
@@ -0,0 +1,77 @@
+---
+sip: 78
+title: iSynth limit reset must not trip circuit breaker
+status: Proposed
+author: Justin J Moses <@justinjmoses>
+discussions-to: https://research.synthetix.io/t/sip-78-isynth-limit-reset-must-not-trip-circuit-breaker/181
+
+created: 2020-08-19
+---
+
+<!--You can leave these HTML comments in your merged SIP and delete the visible duplicate text guides, they will not appear and may be helpful to refer to if you edit it again. This is the suggested template for new SIPs. Note that an SIP number will be assigned by an editor. When opening a pull request to submit your SIP, please use an abbreviated title in the filename, `sip-draft_title_abbrev.md`. The title should be 44 characters or less.-->
+
+## Simple Summary
+
+<!--"If you can't explain it simply, you don't understand it well enough." Simply describe the outcome the proposed changes intends to achieve. This should be non-technical and accessible to a casual community member.-->
+
+Ensure the repricing of an iSynth does not trigger the decentralized circuit breaker.
+
+## Abstract
+
+<!--A short (~200 word) description of the proposed change, the abstract should clearly describe the proposed change. This is what *will* be done if the SIP is implemented, not *why* it should be done or *how* it will be done. If the SIP proposes deploying a new contract, write, "we propose to deploy a new contract that will do x".-->
+
+When the protocolDAO resets an iSynth, it must also reset the pricing for the last exchange price tracker in the `Exchanger` contract.
```

Comment:
Should it reset the pricing to the new `EntryPoint` of the iSynth so that it "starts" on the same level (as discussed).

Reply:
yes- let me rephrase it.